includes/handler/class-delete.php

```diff
@@ -107,9 +120,11 @@ public static function maybe_delete_follower( $activity ) {
 
 		// Verify that Actor is deleted.
 		if ( ! is_wp_error( $follower ) && Tombstone::exists( $activity['actor'] ) ) {
-			Remote_Actors::delete( $follower->ID );
+			$state = Remote_Actors::delete( $follower->ID );
 			self::maybe_delete_interactions( $activity );
 		}
+
+		return $state ?? null;
```

Comment:
```suggestion
		return $state ?? false;
```

I assume `null` would be a failure result? Let's also add `@return` docs to this function